company-profiles/instructure.md

```diff
@@ -0,0 +1,31 @@
+# Instructure
+
+## Company blurb
+
+At Instructure, we believe in the power of people to grow and succeed throughout their lives. Our goal is to amplify that power by creating intuitive products that simplify learning and personal development, facilitate meaningful relationships, and inspire people to go further in their education and careers. We do this by giving smart, creative, passionate people opportunities to create awesome solutions.
+
+## Company size
+
+1000+
+
+## Remote status
+
+*How* you work is much more important than *where* you work. We offer work-from-home flexibility and flex time off ('cause we're all grown-ups here, and we trust our employees to kick some buns and get stuff done).
+
+## Region
+
+Europe
+North America
+Oceania (this includes Australia)
```

Comment:
Readme says worldwide but this has three continents. Which is correct?

Reply:
My apologies, it is those three continents, but I wasn't sure what constitutes worldwide (e.g. must it literally be at every region in the example file?

Reply:
Worldwide would be stated by the company directly. The list tries to be as impartial but accurate as possible but essentially yes, I wouldn't accept worldwide if I know a region was excluded. I wouldn't want someone in India applying coming from this list but being disappointed when the company reject them.

Reply:
Totally makes sense. Thanks for the clarification!